tests/testthat/test-markdown.R

```diff
@@ -570,19 +570,12 @@ test_that("level 1 heading in @details", {
     #' Description.
     #' @details
     #' Leading text goes into details.
-    #' @rawRd
-    #' \\section{This is its own section}{
-    #' \\subsection{Can have a subsection}{
+    #' @section This is its own section:\\subsection{Can have a subsection}{
```

Comment:
Can you put a newline after the `:`?

Reply:
Well, if I want to match the output generated by the markdown, then I can't.  Or we can trim / simplify the output before comparison.

Reply:
Ah ok

Reply:
OK, I'll merge this now then, and will see if #988 is possible without too much refactoring.